packages/compose/src/index.js

```diff
@@ -17,7 +17,7 @@ export { default as useConstrainedTabbing } from './hooks/use-constrained-tabbin
 export { default as useCopyOnClick } from './hooks/use-copy-on-click';
 export { default as useCopyToClipboard } from './hooks/use-copy-to-clipboard';
 export { default as __experimentalUseDialog } from './hooks/use-dialog';
-export { default as __experimentalUseDisabled } from './hooks/use-disabled';
```

Comment:
There was a discussion in https://github.com/WordPress/gutenberg/pull/40162#discussion_r845706144 about leaving the `__experimental` export with a deprecation notice for a moment. It is kind to developers but kind of goes against the idea of __experimental so I'll just echo that here and leave it up to you.

Reply:
I'm going to follow the current guidelines of the project, which is to not do that, "experimental" are "internal" APIs. If experimental are not experimental and can't be changed, it means they have no reason to exist in the first place and I feel strongly that they are very useful to the health of the project, to communicate to developers that we have some internal APIs that we used internally but that you shouldn't use. Potentially at some point, you may use them, but we don't want to commit to backward compatibility yet.

Reply:
I'm going to note as well that I search a-directory and that this one is not used at all.